dcos/dcos-ui
# Human readable quota values

Currently we always show values in megabytes which is not very convenient to read

## Testing

1. Create a quota group with Disk / Memory > 1000 (you can overcommit the cluster asking for bigger quota that it has)
2. Launch a service that consumes the quota in the group
3. check if two places are looking good

## Screenshots

**Before:**
![Screenshot 2019-09-27 at 14 30 14](https://user-images.githubusercontent.com/186223/65769277-80ae5080-e133-11e9-8e94-562104a8cd56.png)
![Screenshot 2019-09-27 at 14 29 58](https://user-images.githubusercontent.com/186223/65769278-80ae5080-e133-11e9-9afb-4d52b27d0378.png)


**After:**
![Screenshot 2019-09-27 at 14 27 38](https://user-images.githubusercontent.com/186223/65769075-fe259100-e132-11e9-8d2f-c793e607d4e4.png)
![Screenshot 2019-09-27 at 14 26 59](https://user-images.githubusercontent.com/186223/65769076-febe2780-e132-11e9-87a0-14f2f7758bc5.png)
